server/datastore/mysql/labels.go

```diff
@@ -15,6 +15,10 @@ import (
 )
 
 func (ds *Datastore) ApplyLabelSpecs(ctx context.Context, specs []*fleet.LabelSpec) (err error) {
+	return ds.ApplyLabelSpecsWithAuthor(ctx, specs, 0)
+}
+
+func (ds *Datastore) ApplyLabelSpecsWithAuthor(ctx context.Context, specs []*fleet.LabelSpec, authorID uint) (err error) {
```

Comment:
There's no `ApplyLabels` method that takes `Label` types as an argument, and there's no reason to add `AuthorID` to `LabelSpec` since it should never be set manually.  So the cleanest way to do this was to update `ApplyLabelSpecs` to `ApplyLabelSpecsWithAuthor` and then create a new `ApplyLabelSpecs` wrapper with a default.  This way we don't have to update the many calls to `ApplyLabelSpecs` in the codebase.